packages/x-date-pickers/src/locales/fiFI.ts

```diff
@@ -42,7 +42,7 @@ const fiFIPickers: Partial<PickersLocaleText> = {
   dateTimePickerToolbarTitle: 'Valitse päivä ja aika',
   timePickerToolbarTitle: 'Valitse aika',
   dateRangePickerToolbarTitle: 'Valitse aikaväli',
-  // timeRangePickerToolbarTitle: 'Select time range',
+  timeRangePickerToolbarTitle: 'Valitse aikaväli',
```

Comment:
Shouldn't the above be `Valitse päivämääräväli` instead? 🤔 
At least based on Google Translate, this equals "Select a time interval", which does not sound completely correct given the English translation of "Select date range". 🤔